src/status_im/ui/screens/wallet/choose_recipient/events.cljs

```diff
@@ -35,16 +35,20 @@
       (when (ethereum/address? s)
         {:address s :chain-id chain-id})))
 
-(defn changed-asset-warning [cofx old-symbol new-symbol]
-  (assoc-in cofx [:db :wallet :send-transaction :asset-error]
-            (i18n/label :t/changed-asset-warning {:old old-symbol :new new-symbol})))
+;; NOTE(janherich) - whenever changing assets, we want to clear the previusly set amount/amount-text
+(defn changed-asset [fx old-symbol new-symbol]
+  (-> fx
+      (assoc-in [:db :wallet :send-transaction :amount] nil)
```

Comment:
might be `(update-in [:db :wallet :send-transaction] dissoc :amount :amount-text)` for first two forms,
or just
```clojure
(update-in fx [:db :wallet :send-transaction] assoc
           :amount nil
           :amount-text nil
           :asset-error (i18n/label :t/changed-asset-warning {:old old-symbol :new new-symbol}))
```